rero_ils/modules/stats/api/pricing.py

```diff
@@ -171,11 +171,16 @@ def number_of_active_patrons(self, library_pid):
         :return: the number of matched active patrons
         :rtype: integer
         """
+        _to = self.to_date.format(fmt="YYYY-MM-DDT23:59:59")
```

Comment:
Can you add a comment to explain why we do not take the default date_range. And better change the variable name.